docs: metric math (#73745)
metabase/metabase: docs/data-modeling/metrics.md

3. Save your changes.

## Explore, compare, and do math with metrics

> You can explore metrics, break them out by different dimensions compare metrics and [measures](../data-studio/measures.md) to each other, and do math with metrics (like add or divide metrics by each other) in the [Metrics Explorer](../questions/metrics-explorer.md).

Reviewer: ```suggestion
To break metrics out by different dimensions, compare metrics and [measures](../data-studio/measures.md), and do math with metrics (like add or divide metrics by each other), check out the [Metrics Explorer](../questions/metrics-explorer.md).
```

 (CountIf([Score] >= 9) - CountIf([Score] <= 6) ) / Count() * 100
```

> Metric formula must be an _aggregation_ of a column, meaning you can have a formula like `Sum([Price])` which will add up all the values in the `Price` column across all records, but you can't have a formula like `[Price] + [Tax]` which adds two values on a per-record basis. If you need to create a "calculated" column, consider using [transforms](../data-studio/transforms/transforms-overview.md) or [models](../data-modeling/models.md) instead.

Reviewer: ```suggestion
A metric formula must be an _aggregation_ of a column, meaning you can have a formula like `Sum([Price])` which will add up all the values in the `Price` column across all records, but you can't have a formula like `[Price] + [Tax]` which adds two values on a per-record basis. If you want to create a "calculated" column, consider creating a [transform](../data-studio/transforms/transforms-overview.md) or [model](../data-modeling/models.md) instead.
```
